src/components/POIForm.component.js

```diff
@@ -187,4 +170,46 @@ class POIForm extends Component {
   }
 }
 
+function FieldGroup({
+  controlId, //indentifier
+  label,
+  inputType, //text vs. textarea vs. file
+  helperText,
```

Comment:
Let's call this `placeholder` so it's consistent with the prop we give `FormControl`